client/keys/list.go

```diff
@@ -1,17 +1,20 @@
 package keys
 
 import (
+	"github.com/cosmos/cosmos-sdk/client"
 	"github.com/spf13/cobra"
 )
 
 func listKeysCmd() *cobra.Command {
-	return &cobra.Command{
+	cmd := &cobra.Command{
 		Use:   "list",
 		Short: "List all keys",
 		Long: `Return a list of all public keys stored by this key manager
 along with their associated name and address.`,
 		RunE: runListCmd,
 	}
+	cmd.Flags().Bool(client.FlagIndentResponse, false, "Add indent to JSON response")
```

Comment:
I think this needs to be done on `keys show` and `keys add` as well

Reply:
OK, I have add it to `keys show` and `keys add`.